dev/droidmate/projects/core/src/main/groovy/org/droidmate/configuration/Configuration.groovy

```diff
@@ -360,10 +360,16 @@ public class Configuration implements IConfiguration
   public List<String> appGuardApisList
 
   /**
-   * Jar with uiautomator-daemon location on the file system. The jar is to be deployed on the android (virtual) device
+   * Jar with uiautomator-daemon location on the file system. The apk is to be deployed on the android (virtual) device
    * to enable GUI actions execution.
    */
-  public Path uiautomatorDaemonJar
+  public Path uiautomatorDaemon
+
+  /**
+   * Jar with uiautomator-daemon-test location on the file system. The apk is to be deployed on the android (virtual) device
```

Comment:
Looks like you forgot to replace first Jar. Can you explain here how these two apks differ? I have no idea. Why do we need both?